src/integration/gen_1_pokemon.py

```diff
@@ -19,12 +20,24 @@ class Pokemon:
     hp: int
     max_hp: int
     level: int
+    attack: int
+    defense: int
+    speed: int
+    special: int
     status: str
     type: str
     move1: str
+    move1_pp: int
+    move1_max_pp: int
```

Comment:
dunno how much you care about clean code but you _could_ put this in a struct
then have an array of moves

don't particularly care for it though

Reply:
Yeah I honestly cba with that ahahaha